src/js/stores/font.js

```diff
@@ -91,6 +91,16 @@ define(function (require, exports, module) {
 
             return fontObj && fontObj.postScriptName;
         },
+        
+        /**
+         * Get the font-family name of a given postScritName. 
+         *
+         * @param {string} postScriptName
+         * @return {?string}
```

Comment:
This doesn't return a string, but an object. Refer to _postScriptMap's JSDoc (boy those are useful)